backend/app/users/sensitive_personal_data/types.py

```diff
@@ -0,0 +1,47 @@
+"""
+This module contains the types used for storing sensitive personal data.
+"""
+
+from enum import Enum
+
+from pydantic import BaseModel, Field
+
+
+class SensitivePersonalDataStatus(Enum):
+    """
+    Represents the status of the sensitive personal data
+    """
+
+    NOT_REQUIRED = "NOT_REQUIRED"
+    REQUIRED = "REQUIRED"
+    CREATED = "CREATED"
+
+
+class CreateSensitivePersonalDataRequest(BaseModel):
+    """
+    Represents the request body for creating sensitive personal data.
+    """
+
+    rsa_key_id: str = Field(description="The key ID of the RSA key used to encrypt the AES key", examples=["key_123"])
+    """
+    The key ID of the RSA key used to encrypt the AES key.
+    """
+
+    aes_encryption_key: str = Field(description="RSA encrypted AES key used to encrypt/decrypt the sensitive user data",
+                                    examples=["ZGFzZGE="])
+    """
+    The AES key used to encrypt/decrypt the sensitive user data. it is RSA encrypted.
+    """
+
+    aes_encrypted_data: str = Field(description="the encrypted user sensitive  data joined with iv",
```

Comment:
max max size, this depends on the size of the data that will be encrypted, 
lets assume 15 fields with keys (first, last, gender, age, country, city, street, email, tel, employment status,...) each 2* 256 char (256 the json key and 256 the value) long each char 3 bytes (worst case for utf-8) Total AES-CGP 256-128tag =P+Tag size+IVsize = P+16+12bytes
=15*3*2*256 + 16 +12 bytes =23068bytes then convered to base64 approx *4/3 = 30758 we can say 35000 to be safe